app/infrastructure/persistence/repositories/chain_repair_log_repo.py

```
@@ -0,0 +1,68 @@
+        self,
+        *,
+        tenant_id: str,
+        epoch_id: str,
+        break_at_event_seq: int,
+        break_reason: str,
+        repair_initiated_by: str,
+        approval_required: bool,
+        repair_reference: str | None,
+    ) -> ChainRepairLog:
+        """Insert a new chain_repair_log row and return it."""
+        now = datetime.now(timezone.utc)
```

Ruff (0.15.5)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)